packages/backend/ubuntu-pro.go

```diff
@@ -0,0 +1,163 @@
+package main
+import (
+    "os/exec"
+    "context"
+    "errors"
+    pb "github.com/canonical/desktop-security-center/packages/proto"
+    "google.golang.org/protobuf/types/known/emptypb"
+    "github.com/tidwall/gjson"
+    "github.com/godbus/dbus/v5"
+)
+
+var conn *dbus.Conn
```

Comment:
On a similar note to the comment about using a new function, I would look at defining a struct that holds all your service variables such as this dbus connection. Even if you are not currently passing in a dbus conn to the service, I would define a new func that does the setup of establishing the connection and then holding it for use by functions in this service

For example:
[Defining a struct that holds different variables relevant to the serivce](https://github.com/matthew-hagemann/ubuntu-desktop-provision/blob/main/provd/internal/services/user/user.go#L30-L37)
[New function that initializes all the variables.](https://github.com/matthew-hagemann/ubuntu-desktop-provision/blob/main/provd/internal/services/user/user.go#L44-L76)
I would also use it to hold dbus objects you, see next comment!